### ISSUE TITLE
[Error] Server failed to synchronize recipe data with the client

### ISSUE THREAD

**Issue by BadPizza23** [QuantumXenon/origins-plus-plus#140]
I can't use the bug report layout (Coz I dunno the answers) I'm using the curse forge version- on overwolf and getting this message

"server failed to synchronize recipe data with the client"
when my friend tries to join! :(

**Comment by QuantumXenon** [comment]
I've not seen that error before. Can you provide logs?

**Comment by BadPizza23** [comment]
I sadly cannot, I was using a LAN, and I have NO idea how to get a log! I am very sorry!

**Comment by MrKudoman** [comment]
i am also having that error, i however have a log 

[latest - Copy.log](https://github.com/QuantumXenon/origins-plus-plus/files/10636773/latest.-.Copy.log)

**Comment by MrKudoman** [comment]
I've even tried rolling back versions of the modpack to other releases and the error persists, wondering what the issue may be

**Comment by QuantumXenon** [comment]
@MrKudoman Are you also trying to open the server to LAN?

**Comment by QuantumXenon** [comment]
@BadPizza23 @MrKudoman I've looked at the logs, and it appears to be an issue with Chipped:
```
[20:36:50] [Netty Server IO #1/ERROR]: Error receiving packet 106
java.lang.NoSuchMethodError: 'void net.minecraft.class_2540.method_34062(java.util.Collection, java.util.function.BiConsumer)'
	at Not Enough Crashes deobfuscated stack trace.(1.19.2+build.28) ~[?:?]
	at earth.terrarium.chipped.recipe.ChippedRecipe$Serializer.toNetwork(ChippedRecipe.java:146) ~[chipped-2.0.0-fabric.jar:?]
	at earth.terrarium.chipped.recipe.ChippedRecipe$Serializer.write(ChippedRecipe.java:104) ~[chipped-2.0.0-fabric.jar:?]
	at net.minecraft.network.packet.s2c.play.SynchronizeRecipesS2CPacket.writeRecipe(SynchronizeRecipesS2CPacket:52) ~[?:?]
	at net.minecraft.network.PacketByteBuf.writeCollection(PacketByteBuf:179) ~[?:?]
	at net.minecraft.network.packet.s2c.play.SynchronizeRecipesS2CPacket.write(SynchronizeRecipesS2CPacket:27) ~[?:?]
	at net.minecraft.network.PacketEncoder.encode(PacketEncoder:45) ~[?:?]
	at net.minecraft.network.PacketEncoder.encode(PacketEncoder:14) ~[?:?]
	at io.netty.handler.codec.MessageToByteEncoder.write(MessageToByteEncoder.java:107) [netty-codec-4.1.77.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWrite0(AbstractChannelHandlerContext.java:717) [netty-transport-4.1.77.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext.invokeWriteAndFlush(AbstractChannelHandlerContext.java:764) [netty-transport-4.1.77.Final.jar:?]
	at io.netty.channel.AbstractChannelHandlerContext$WriteTask.run(AbstractChannelHandlerContext.java:1071) [netty-transport-4.1.77.Final.jar:?]
	at io.netty.util.concurrent.AbstractEventExecutor.runTask(AbstractEventExecutor.java:174) [netty-common-4.1.77.Final.jar:?]
	at io.netty.util.concurrent.AbstractEventExecutor.safeExecute(AbstractEventExecutor.java:167) [netty-common-4.1.77.Final.jar:?]
	at io.netty.util.concurrent.SingleThreadEventExecutor.runAllTasks(SingleThreadEventExecutor.java:470) [netty-common-4.1.77.Final.jar:?]
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:503) [netty-transport-4.1.77.Final.jar:?]
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:995) [netty-common-4.1.77.Final.jar:?]
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74) [netty-common-4.1.77.Final.jar:?]
	at java.lang.Thread.run(Thread.java:833) [?:?]
```

**Comment by MrKudoman** [comment]
I am not attempting to do LAN, I’m just trying to run a regular server off my pc for friends to join, and I’ll try disabling chipped tonight! Thanks for helping!

**Comment by MrKudoman** [comment]
Disabling chipped has worked perfectly for us! thanks for the help

**Comment by QuantumXenon** [comment]
@MrKudoman Great! Hopefully the issue is fixed soon. Does that work for you @BadPizza23?

**Comment by QuantumXenon** [comment]
@BadPizza23 @MrKudoman Try updating the modpack - Chipped has released an update which is specifically for 1.19.2, so perhaps the error you had has been fixed.
